Description: Update icu pkgconfig module lookup
 After upgrading to icu76 the build started failing to look up symbols
 from icu when using the --system-icu option. Updating the pkgconfig 
 lookups to also read the icu-uc pkgconfig fixes the build. 
Origin: https://github.com/mozilla-firefox/firefox/commit/212f5ffea7a147d43447df0e544749b3bb750afc.patch

diff --git a/js/moz.configure b/js/moz.configure
index 3c3d0d4359f0..3f095f5f4851 100644
--- a/js/moz.configure
+++ b/js/moz.configure
@@ -959,7 +959,9 @@ set_define(
 # ======================================================
 option("--with-system-icu", help="Use system ICU")
 
-system_icu = pkg_check_modules("MOZ_ICU", "icu-i18n >= 69.1", when="--with-system-icu")
+system_icu = pkg_check_modules(
+    "MOZ_ICU", "icu-uc icu-i18n >= 69.1", when="--with-system-icu"
+)
 
 set_config("MOZ_SYSTEM_ICU", True, when=system_icu)
 set_define("MOZ_SYSTEM_ICU", True, when=system_icu)
